packages/terra-tabs/translations/en.json

```
@@ -1,8 +1,14 @@
 {
   "Terra.tabs.more": "More",
+  "Terra.tabs.hint.tabList": "Contains {numberOfTabs} items",
+  "Terra.tabs.hint.hiddenTabs": "{hiddenTabs} items are not displayed",
+  "Terra.tabs.hint.tabMenu": "{hiddenTabs} items are hidden. Select to display {hiddenTabs} hidden items",
+  "Terra.tabs.hint.removable":". item is removable via Delete or Back Space key",
+  "Terra.tabs.hint.currentTabClosed":" Removed",
```

```suggestion
  "Terra.tabs.hint.removable":"item is removable via Delete or Back Space key",
  "Terra.tabs.hint.currentTabClosed":"Removed",
```
`.` and empty space needs to be handled while consuming the translations strings instead of having them on translation file.

---

Done https://github.com/cerner/terra-framework/commit/628052153b88461ef88733be6b05c1fb519e3f41